### ISSUE TITLE
No cameras found

### ISSUE THREAD

**Issue by nomis5000** [datazuul/eazy-bookscanner#5]
I always get the message: "No cameras found. Please attach two cameras before retrying again." I'm using arch linux and two Ixus 160 cameras are attached, both have the latest chdkptp on them. With chdkptp I see them both and can control them. Do you have any idea?

**Comment by datazuul** [comment]
thank you for your test. to get the camera working, a new class has to be added in this project:
https://github.com/datazuul/CHDK-PTP-Java/tree/master/src/main/java/chdk/ptp/java/camera

And then be added to this list:
https://github.com/datazuul/CHDK-PTP-Java/blob/master/src/main/java/chdk/ptp/java/SupportedCamera.java

Could you please post here the productID of your camera? Is it `32aa  Powershot ELPH 160 / IXUS 160`?

**Comment by nomis5000** [comment]
Thank you for your reply! Yes, that is my camera. I would try things on my own, but I have not really experiences with java and did already fail by compiling the actual project with maven - sorry!

**Comment by datazuul** [comment]
I've added a basic class file for your camera to CHDK-PTP-Java (see above) and compiled a new eazy-bookscanner version 0.2.1-SNAPSHOT you can download here: https://github.com/datazuul/eazy-bookscanner/blob/main/dist/eazy-bookscanner-app-0.2.1-SNAPSHOT.zip

Camera should now be detected. Please test. As camera focus mode is set by executing clicks you would have to do manually the setFocusMode-method for your camera is not implemented as I do not know what click sequence is needed. Please report (or implement in a fork) the click sequences. Have a lokk here for an example: https://github.com/datazuul/CHDK-PTP-Java/blob/87b86192e9dc0aa0347c775cec236046bcd42c85/src/main/java/chdk/ptp/java/camera/SX160ISCamera.java#L37

**Comment by datazuul** [comment]
any success with new version? are the cameras at least detected?

**Comment by nomis5000** [comment]
Yes! And I'm very sorry for the late reply. And also for my low skills. The cameras was detected and I was able to take some photos. Sometimes one camera was lost, so I had to restart. But this could be a problem of the camera, because with piscan I had the same problem. I failed totally to understand the part with the click sequence. I will try that another time, but the next weeks and months are to many other things to do. If I had seen this coming, I would not have wasted your time.
